doc/man1/s_client.pod

```diff
@@ -473,59 +473,57 @@ will be used.
 
 =item B<-tlsextdebug>
 
-print out a hex dump of any TLS extensions received from the server.
+Print out a hex dump of any TLS extensions received from the server.
 
 =item B<-no_ticket>
 
-disable RFC4507bis session ticket support.
+Disable RFC4507bis session ticket support.
 
 =item B<-sess_out filename>
 
-output SSL session to B<filename>
+Output SSL session to B<filename>.
 
 =item B<-sess_in sess.pem>
 
-load SSL session from B<filename>. The client will attempt to resume a
+Load SSL session from B<filename>. The client will attempt to resume a
 connection from this session.
 
 =item B<-engine id>
 
-specifying an engine (by its unique B<id> string) will cause B<s_client>
+Specifying an engine (by its unique B<id> string) will cause B<s_client>
 to attempt to obtain a functional reference to the specified engine,
 thus initialising it if needed. The engine will then be set as the default
 for all available algorithms.
 
 =item B<-rand file(s)>
 
-a file or files containing random data used to seed the random number
+A file or files containing random data used to seed the random number
 generator, or an EGD socket (see L<RAND_egd(3)>).
 Multiple files can be specified separated by an OS-dependent character.
 The separator is B<;> for MS-Windows, B<,> for OpenVMS, and B<:> for
 all others.
 
 =item B<-serverinfo types>
 
-a list of comma-separated TLS Extension Types (numbers between 0 and
+A list of comma-separated TLS Extension Types (numbers between 0 and
 65535).  Each type will be sent as an empty ClientHello TLS Extension.
 The server's response (if any) will be encoded and displayed as a PEM
 file.
 
 =item B<-status>
 
-sends a certificate status request to the server (OCSP stapling). The server
+Sends a certificate status request to the server (OCSP stapling). The server
 response (if any) is printed out.
 
 =item B<-alpn protocols>, B<-nextprotoneg protocols>
 
-these flags enable the 
-Enable the Application-Layer Protocol Negotiation or Next Protocol
-Negotiation extension, respectively. ALPN is the IETF standard and
-replaces NPN.
-The B<protocols> list is a
-comma-separated protocol names that the client should advertise
-support for. The list should contain most wanted protocols first.
-Protocol names are printable ASCII strings, for example "http/1.1" or
-"spdy/3".
+These flags enable the Enable the Application-Layer Protocol Negotiation
+or Next Protocol Negotiation extension, respectively. ALPN is the IETF
+standard and replaces NPN.
+The B<protocols> list is a comma-separated protocol names that the
+client should advertise support for. The list should contain most wanted
```

Comment:
This is a re-wrapping of the text without changes.  I did think about dropping the "Enable the".

The second sentence is bad.  This text is repeated in s_server.pod.